com.unity.render-pipelines.core/Editor/Lighting/ProbeVolume/ProbeVolumeUI.Drawer.cs

```diff
@@ -138,23 +138,47 @@ static void Drawer_VolumeContent(SerializedProbeVolume serialized, Editor owner)
-            EditorGUI.BeginProperty(rect, Styles.s_MinMaxSubdivSlider, serialized.maxSubdivisionMultiplier);
 
             // Round min and max subdiv
-            float maxSubdiv = ProbeReferenceVolume.instance.GetMaxSubdivision(1) - 1;
-            float min = Mathf.Round(serialized.minSubdivisionMultiplier.floatValue * maxSubdiv) / maxSubdiv;
-            float max = Mathf.Round(serialized.maxSubdivisionMultiplier.floatValue * maxSubdiv) / maxSubdiv;
-
-            EditorGUILayout.MinMaxSlider(Styles.s_MinMaxSubdivSlider, ref min, ref max, 0, 1);
-            serialized.minSubdivisionMultiplier.floatValue = Mathf.Max(0.00f, min);
-            serialized.maxSubdivisionMultiplier.floatValue = Mathf.Max(0.01f, max);
-            EditorGUI.EndProperty();
-            EditorGUI.EndProperty();
-
-            int minSubdivInVolume = ProbeReferenceVolume.instance.GetMaxSubdivision(1 - serialized.minSubdivisionMultiplier.floatValue);
-            int maxSubdivInVolume = ProbeReferenceVolume.instance.GetMaxSubdivision(1 - serialized.maxSubdivisionMultiplier.floatValue);
-            EditorGUILayout.HelpBox($"The distance between probes will fluctuate between : {ProbeReferenceVolume.instance.GetDistanceBetweenProbes(maxSubdivInVolume)}m and {ProbeReferenceVolume.instance.GetDistanceBetweenProbes(minSubdivInVolume)}m", MessageType.Info);
+            int maxSubdiv = ProbeReferenceVolume.instance.GetMaxSubdivision() - 1;
+            // it's likely we don't have a profile loaded yet.
+            if (maxSubdiv < 0)
+            {
+                if (ProbeReferenceVolume.instance.sceneData != null)
+                {
+                    ProbeVolume pv = (serialized.serializedObject.targetObject as ProbeVolume);
+
+                    var profile = ProbeReferenceVolume.instance.sceneData.GetProfileForScene(pv.gameObject.scene);
+
+                    if (profile != null)
+                    {
+                        ProbeReferenceVolume.instance.SetMinBrickAndMaxSubdiv(profile.minBrickSize, profile.maxSubdivision);
+                        maxSubdiv = ProbeReferenceVolume.instance.GetMaxSubdivision() - 1;
+                    }
+                    else
+                    {
+                        maxSubdiv = 0;
+                    }
+                }
+            }
+
+            EditorGUILayout.LabelField("Subdivision Overrides", EditorStyles.boldLabel);
+            EditorGUI.indentLevel++;
+            int value = serialized.highestSubdivisionLevelOverride.intValue;
+
+            // We were initialized, but we cannot know the highest subdiv statically, so we need to resort to this.
+            if (serialized.highestSubdivisionLevelOverride.intValue < 0)
+                serialized.highestSubdivisionLevelOverride.intValue = maxSubdiv;
+
+            serialized.highestSubdivisionLevelOverride.intValue = EditorGUILayout.IntSlider("Highest Subdivision Level", serialized.highestSubdivisionLevelOverride.intValue, 0, maxSubdiv);
```

Comment:
Missing `EditorGUI.BeginProperty` and EndProperty to correctly handle presets, prefabs, undo, etc...
(you can take the code before your PR as a reference to add it)